pkg/tasks/monitoring_plugin.go

```diff
@@ -37,6 +38,19 @@ func NewMonitoringPluginTask(client *client.Client, factory *manifests.Factory,
 }
 
 func (t *MonitoringPluginTask) Run(ctx context.Context) error {
+	// ensure console is enabled and bail out if it isn't
```

Comment:
Maybe worth adding a comment that the console capability (like other capabilities) can only go from disabled -> enabled and not the other way around, meaning that CMO doesn't have to deal with removal of the console plugin resources.

Reply:
Are we okay with doing this in a follow-up patch, so that this blocker is resolved?

Reply:
Scratch, that, I have added the change in this PR since I had to address a comment.